sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f

```diff
@@ -205,6 +205,7 @@ SUBROUTINE CALRAD_WCLOUD
   logical post_abig16, post_abig17, post_abigr ! if true, user requested at least one abi channel
   logical fix_abig16, fix_abig17   ! if true, abi_g16, abi_g17 fix files are available
   logical post_ahi8 ! if true, user requested at least on ahi channel (himawari8)
+  logical post_ssmis17 ! if true, user requested at least on ssmis_f17 channel
```

Comment:
@LinZhu-NOAA Can you add a change log with your name next to the line 14 in this routine? Thanks!

Reply:
@WenMeng-NOAA The change log is added, please check. Thanks!